config/locales/en/course/assessment/submission/submissions.yml

```diff
@@ -11,6 +11,7 @@ en:
             THIS ACTION IS IRREVERSIBLE
             
             Are you sure you want to submit? You will no longer be able to amend your submission!
+          confirm_unsubmit: 'Are you sure you want to unsubmit?'
```

Comment:
Let's change the message to the v1 one:
```
This will permit the student to change his submission. 

NOTE THAT YOU CANNOT UNDO THIS!! 

Only the student can decide when to submit his/her work once it is unsubmitted. Are you sure you want to proceed?
```

This will tell the tutor what does the button does for the first time.